src/Sit.scala

```diff
@@ -22,6 +22,14 @@ class Sit(private val projectPath: String,
    */
   private def persisToDisk: Unit = {
     // TODO
+    val fw = new FileWriter(sitConfigPath, true)
+    val uuid: UUID = randomUUID()
+    val form = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
+    val now = form.format(Calendar.getInstance().getTime)
+
+    val commitInfo: String = s"[$now] ${uuid.toString}\n"

```

Comment:
We should persist the whole commit linked list to .sit

But I am OK if you want to merge this one first.